graf3d/eve7/src/REveTypes.cxx

```diff
@@ -43,7 +43,7 @@ REveException REX::operator+(const REveException &s1,  const char *s2)
 REveException REX::operator+(const REveException &s1, ElementId_t x)
 { REveException r(s1); r.append(std::to_string(x)); return r; }
 
-REX::RLogChannel &REX::EveLog() {
+REX::RLogChannel &REX::REveLog() {
    static RLogChannel sLog("ROOT.Eve");
    return sLog;
 }
```

Comment:
Also here missing new line

Reply:
@linev This is automatic alignment by Visual Studio Code editor. I'm checking how to change the editor default settings.